openmc/filter_expansion.py

```diff
@@ -463,3 +463,65 @@ def to_xml_element(self):
         subelement.text = str(self.r)
 
         return element
+
+
+class ZernikeRadialFilter(ZernikeFilter):
+    r"""Score radial Zernike expansion moments in space up to specified order.
+
+    The Zernike polynomials are defined the same as in ZernikeFilter.
+
+    .. math::
+        Z_n^m(\rho, \theta) = R_n^m(\rho) \cos (m\theta), \quad m > 0
+
+        Z_n^{m}(\rho, \theta) = R_n^{m}(\rho) \sin (m\theta), \quad m < 0
+
+        Z_n^{m}(\rho, \theta) = R_n^{m}(\rho), \quad m = 0
```

Comment:
Go ahead and remove the two cases above for `m > 0` and `m < 0`.  Maybe substitute in 0 for m in the remaining case (e.g. change `Z_n^{m}` to `Z_n^0`).